src/ra_server.erl

```diff
@@ -2518,6 +2486,15 @@ fetch_term(Idx, #{log := Log0} = State) ->
             {Term, State#{log => Log}}
     end.
 
+
+transform_for_partial_read(_Idx, _Term, {'$usr', _, Cmd, _}) ->
+    %% stip usr commands of ra internals
```

Comment:
```suggestion
    %% strip usr commands of ra internals
```

also I would suggest moving the comment on L2166 down as a sort of docstring for this function, I think that's a good description that would fit better right next to the function